test-framework/core/src/main/java/org/keycloak/test/framework/config/Config.java

```diff
@@ -33,7 +41,8 @@ public static SmallRyeConfig initConfig() {
         SmallRyeConfigBuilder configBuilder = new SmallRyeConfigBuilder()
                 .addDefaultSources()
                 .addDefaultInterceptors()
-                .withSources(new DotEnvConfigSourceProvider());
```

Comment:
@stianst Do you plan to preserve this class?

Reply:
Not quite sure what you mean about preserve this class. DotEnvConfigSourceProvider comes from SmallRye, but actually it's part of the default sources that are added by `addDefaultSources()`.

Reply:
I initially thought it's implemented by us in the PoC.